nix/hydra/project.nix

```diff
@@ -53,33 +53,49 @@ let
 
     inputMap = { "https://input-output-hk.github.io/cardano-haskell-packages" = CHaP; };
 
-    modules = [
-      # Strip debugging symbols from exes (smaller closures)
-      {
-        packages.hydra-node.dontStrip = false;
-        packages.hydra-tui.dontStrip = false;
-        packages.hydraw.dontStrip = false;
-      }
-      # Avoid plutus-tx errors in haddock (see also cabal.project)
-      {
-        packages.hydra-plutus.setupHaddockFlags = [ "--ghc-options='-fplugin-opt PlutusTx.Plugin:defer-errors'" ];
-        packages.plutus-merkle-tree.setupHaddockFlags = [ "--ghc-options='-fplugin-opt PlutusTx.Plugin:defer-errors'" ];
-        packages.plutus-cbor.setupHaddockFlags = [ "--ghc-options='-fplugin-opt PlutusTx.Plugin:defer-errors'" ];
-      }
-      # Fix compliation of strict-containers (see also cabal.project)
-      {
-        packages.strict-containers.ghcOptions = [ "-Wno-noncanonical-monad-instances" ];
+    modules = [{
+      packages = {
+
+        # Strip debugging symbols from exes (smaller closures)
+        hydra-node.dontStrip = false;
+        hydra-tui.dontStrip = false;
+        hydraw.dontStrip = false;
+
+        # Avoid plutus-tx errors in haddock (see also cabal.project)
+        hydra-plutus.setupHaddockFlags = [ "--ghc-options='-fplugin-opt PlutusTx.Plugin:defer-errors'" ];
+
+        plutus-merkle-tree.setupHaddockFlags = [ "--ghc-options='-fplugin-opt PlutusTx.Plugin:defer-errors'" ];
+        plutus-cbor.setupHaddockFlags = [ "--ghc-options='-fplugin-opt PlutusTx.Plugin:defer-errors'" ];
+
+        # Fix compliation of strict-containers (see also cabal.project)
+        strict-containers.ghcOptions = [ "-Wno-noncanonical-monad-instances" ];
+
         # XXX: Could not figure out where to make this flag ^^^ effective in the haddock build
-        packages.strict-containers.doHaddock = false;
-      }
+        strict-containers.doHaddock = false;
+
+        # -Werror for CI
```

Comment:
This is not only enabling Werror for CI but for any `nix build` involving these packages. Either we put `-Werror` somewhere else or update this comment.